lib/internal/util.js

```diff
@@ -320,14 +321,29 @@ const kCustomPromisifiedSymbol = SymbolFor('nodejs.util.promisify.custom');
 const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
 
 let validateFunction;
+let validateInteger;
 
-function promisify(original) {
+function promisify(original, options) {
```

Comment:
```suggestion
function promisify(original, options = ObjectCreate(null)) {
```
or at least
```suggestion
function promisify(original, options = undefined) {
```

Reply:
As it is for now, it won't handle `option === null`. Or if it implies validation&throwing afterwards, we can't do it to keep accidental second parameter working as default.

Explicit `undefined` wouldn't hurt though.